code/network/multi_observer.cpp

```diff
@@ -59,7 +59,7 @@ void multi_obs_create_player(int player_num,char *name,net_addr *addr,player *pl
 	Net_players[player_num].last_heard_time = timer_get_fixed_seconds();
```

Comment:
Looks like `last_heard_time` could be converted to `TIMESTAMP` or `UI_TIMESTAMP`

Reply:
Ok.